backend/api/serializers.py

```diff
@@ -11,4 +11,9 @@
 class PostSerializer(serializers.ModelSerializer):
   class Meta:
     model = post.Post
-    fields = ['title', 'author_id', 'description', 'content', 'contentType', 'visibility']
\ No newline at end of file
+    fields = ['title', 'author_id', 'description', 'content', 'contentType', 'visibility']
+
+class AuthorSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = author.Author
+    fields = ['username','password','git_url']
```

Comment:
The model should be signup_request, since new signups are sent to signup_request waiting for acceptation.

Reply:
Author cannot get add unless it is approved by server admin. 

This indicates the model you should look at is author sign up request